code/user_settings.py

```diff
@@ -13,37 +11,33 @@
 if not SETTINGS_DIR.is_dir():
     os.mkdir(SETTINGS_DIR)
 
-mod = Module()
-ctx = Context()
-
 
-def _load_csv_dict(
-    file_name: str, headers=Tuple[str, str], default: Dict[str, str] = {}
-) -> Dict[str, str]:
-    """Load a word mapping from a CSV file. If it doesn't exist, create it."""
-    assert file_name.endswith(".csv")
-    path = SETTINGS_DIR / file_name
+def get_list_from_csv(
+    filename: str, headers: Tuple[str, str], default: Dict[str, str] = {}
+):
+    """Retrieves list from CSV"""
+    path = SETTINGS_DIR / filename
+    assert filename.endswith(".csv")
 
-    # Create the file if it doesn't exist
-    if not SETTINGS_DIR.is_dir():
```

Comment:
Is there a reason this code to auto-create the settings directory got dropped? Seems like we'll need this for first-time users.

Reply:
It's there:
https://github.com/knausj85/knausj_talon/blob/50f55ef7abb829421cc494fae0a8bdeb3688f8df/code/user_settings.py#L11